openfl-workspace/keras_nlp/src/dataloader_utils.py

```diff
@@ -1,4 +1,5 @@
-"""Copyright (C) 2020-2021 Intel Corporation.
+"""Copyright (C) 2020-2021 Intel Corporation
+   SPDX-License-Identifier: Apache-2.0
 
 Licensed subject to the terms of the separately executed evaluation
 license agreement between Intel Corporation and you.
```

Comment:
Should we include full license text in checker rules in order to use only one type of file header?

Reply:
@grib0ed0v could you please clarify this?

Reply:
@itrushkin  @dmitryagapov @ViktoriiaRomanova it's okay to keep only license header, not a full text.